### PR TITLE
Allow emoji or image after the entry description

### LINKED ISSUE #60: Allow emoji or image after the entry description
```
- [Title](URL) - Description. ⭐
```

or

```
- [Title](URL) - Description. ![](image.png)
```

or

```
- [Title](URL) - Description. <img src="image.png">
```

### PR DESCRIPTION
**Pull request by chinesedfan**
Fixes #60.

In fact, emoji was allowed after the entry description by #29.

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Overall, looks really solid. See my comment in #60 as Sindre has the final call on all formatting decisions like this.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-lint/issues/60#issuecomment-478368459

File: lib/trim-emoji.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const emojiRegex = require('emoji-regex');
+
+module.exports = text => {
+	let match;
+	let regex = emojiRegex();
+	while ((match = regex.exec(text))) {
+		const emoji = match[0];
+		const codePoints = [...emoji].length;
+		if (match.index === 0) {
+			text = text.substr(codePoints).trim();
```
Comment (sindresorhus) [inline lib/trim-emoji.js]: I would prefer using `slice` instead of `substr`.

File: lib/trim-emoji.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+			regex = emojiRegex();
+		} else if (match.index + codePoints === text.length) {
+			text = text.substr(0, match.index).trim();
+			regex = emojiRegex();
+		}
+	}
+
+	return text;
```
Comment (sindresorhus) [inline lib/trim-emoji.js]: Why not just `text.replace(emojiRegex, '')`?

Comment (chinesedfan) [reply lib/trim-emoji.js]: See my test cases. I want to trim emojis and spaces, but **NOT** to remove inner emojis and **NOT** to trim spaces if no emojis.

- t.deepEqual(trimEmoji('⭐ hello'), 'hello');
- t.deepEqual(trimEmoji(' hello '), ' hello ');
- t.deepEqual(trimEmoji('hello⭐hello'), 'hello⭐hello');

File: lib/trim-emoji.js (hunk 1 above)
Comment (chinesedfan) [reply lib/trim-emoji.js]: Sure. But sadly you rejected this feature, https://github.com/sindresorhus/awesome-lint/issues/60#issuecomment-478368459.

**Comment by transitive-bullshit** [comment]
@chinesedfan can you update this PR to reflect the decision mad in #60?

Thanks!

**Comment by chinesedfan** [comment]
@transitive-bullshit Updated.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**